### ISSUE TITLE
Cannot get the result as yours ...

### ISSUE THREAD

**Issue by ckl8964** [llSourcell/How-to-Predict-Stock-Prices-Easily-Demo#4]
Hi,

I run this demo code in my ubuntu 16.04.2 installed with tensorflow 1.0, keras 2.0.2 and python 2.7.12. Everything runs fine without error and warning, all console printout are similar to yours. However, the output prediction diagram is quite different from yours.

All those prediction lines are just pointing upwards of nearly the same slope. 

What's wrong with it ? Thanks.

**Comment by Steviey** [comment]
I got the same effect on Windows. Then I deleted all cached folders in project root, extracted the source files again and clicked "Kernel > Restart & Run All" in the browser. The effect disappeared.
Since I made some other modifications, not 100% sure whether this was the real fix.

**Comment by ckl8964** [comment]
I deleted the only intermediate files lstm.pyc and restarted the linux to run it again. I found the result was different every time !! Would you mind to share what modification you made for it ? Thanks.

**Comment by Steviey** [comment]
It's all here --> https://github.com/llSourcell/How-to-Predict-Stock-Prices-Easily-Demo/issues/3 (Chapter: Update/Solution)

**Comment by ckl8964** [comment]
ummm ... it is the problem related to running under Windows and python version 3.x. 

I run it under linux and python 2.7 which should be perfect to those codes.

**Comment by Steviey** [comment]
I got it again and illustrate the effect here: https://github.com/llSourcell/How-to-Predict-Stock-Prices-Easily-Demo/issues/3

**Comment by ckl8964** [comment]
Yes, we got similar results.... hope someone could  help ..

**Comment by Steviey** [comment]
There is a possible explanation about it. The feed is randomly shuffled. Take a look at our win.-thread.
